src/test/blockchain_tests.cpp

```diff
@@ -1,9 +1,9 @@
 #include <boost/test/unit_test.hpp>
 
-#include "stdlib.h"
+#include <stdlib.h>
 
-#include "rpc/blockchain.h"
-#include "test/test_bitcoin.h"
+#include <rpc/blockchain.cpp>
```

Comment:
This seems incorrect

Reply:
It include `blockchain.h` previously, but you include `blockchain.cpp`

Reply:
Ouch, now fixed! Thanks for catching this. I missed the `.cpp` extension!